src/status_im/ui/screens/chat/stickers/views.cljs

```diff
@@ -27,33 +35,65 @@
 
-(defn- stickers-panel [stickers]
-  [react/scroll-view {:style {:flex 1} :condtent-container-style {:flex 1}}
-   [react/view {:style styles/stickers-panel}
-    (for [{:keys [uri] :as sticker} stickers]
-      ^{:key uri}
-      [react/touchable-highlight {:style    {:height 75 :width 75 :margin 5}
-                                  :on-press #(re-frame/dispatch [:chat/send-sticker sticker])}
-       [react/image {:style {:resize-mode :cover :width "100%" :height "100%"} :source {:uri uri}}]])]])
+(defview stickers-panel [stickers window-width]
+  [react/view {:width window-width :flex 1}
+   [react/scroll-view
+    [react/view {:style styles/stickers-panel}
+     (for [{:keys [uri] :as sticker} stickers]
+       ^{:key uri}
+       [react/touchable-highlight {:style    {:height 75 :width 75 :margin 5}
+                                   :on-press #(re-frame/dispatch [:chat/send-sticker sticker])}
+        [react/image {:style {:resize-mode :cover :width "100%" :height "100%"} :source {:uri uri}}]])]]])
 
-(defview recent-stickers-panel []
+(defview recent-stickers-panel [window-width]
   (letsubs [stickers [:stickers/recent]]
     (if (seq stickers)
-      [stickers-panel stickers]
-      [react/view {:style {:flex 1 :align-items :center :justify-content :center}}
+      [stickers-panel stickers window-width]
+      [react/view {:style {:flex 1 :align-items :center :justify-content :center :width window-width}}
        [vector-icons/icon :stickers-icons/stickers-big {:color colors/gray}]
        [react/text {:style {:margin-top 8 :font-size 17}} (i18n/label :t/recently-used-stickers)]])))
 
-(def icon-size 28)
+(defn update-scroll-position [ref installed-packs selected-pack window-width]
+  (when ref
+    (let [x (if (= selected-pack :recent)
+              0
+              (* (inc (some #(when (= selected-pack (:id (second %))) (first %))
+                            (map-indexed vector installed-packs)))
+                 window-width))]
+      (.scrollTo ref (clj->js {:x x :animated true})))))
 
-(defn pack-icon [{:keys [id on-press selected? background-color]
+(defn on-scroll [e installed-packs window-width]
+  (let [num     (/ (.-nativeEvent.contentOffset.x e) window-width)
+        pack-id (if (zero? num)
+                  :recent
+                  (get-in (vec installed-packs) [(dec num) :id]))]
+    (when pack-id
+      (re-frame/dispatch [:stickers/select-pack pack-id]))))
+
+(defview stickers-paging-panel [installed-packs selected-pack]
+  (letsubs [ref          (atom nil)
+            window-width [:dimensions/window-width]]
+    {:component-will-update (fn [_ [_ installed-packs selected-pack]]
+                              (update-scroll-position @ref installed-packs selected-pack window-width))
+     :component-did-mount   #(update-scroll-position @ref installed-packs selected-pack window-width)}
+    [react/scroll-view {:style                             {:flex 1} :horizontal true :paging-enabled true
```

Comment:
maybe move down `:horizontal` and `:paging-enabled` if they are not related to `:style`